### PR TITLE
Adding Tree Sort in Java

### PR DESCRIPTION
**Pull request by Khushi2402**

### CODE REVIEW

File: Java/Sorting Algorithm/TreeSort.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
+public class TreeSort {
```
Comment (ShehrozIrfan) [inline Java/Sorting Algorithm/TreeSort.java:3]: Just a minor:
Add the comment on the top of the file:
`Tree Sort Implementation` or something like this.

**Review by ShehrozIrfan — CHANGES_REQUESTED** [review]
Good work.
Just need to add comment on the top of file.

File: Java/Sorting Algorithm/TreeSort.java (hunk 1 above)
Comment (Khushi2402) [reply Java/Sorting Algorithm/TreeSort.java:3]: Done

**Review by ShehrozIrfan — APPROVED** [review]
LGTM 🎉

Thank you for contribution 🚀

**Pull request merged**